runtime/doc/options.txt

```diff
@@ -4006,6 +4005,16 @@ A jump table for the options with a short description can be found at |Q_op|.
 	"precedes".  "SpecialKey" for "nbsp", "space", "tab" and "trail".
 	|hl-NonText| |hl-SpecialKey|
 
+						*'incsubstitute'* *'ics'*
+'incsubstitute' 'ics'	number	(default 0)
+			global
+			
+	If set to a nonzero value, substitutions (|:s|) are updated live while
+	the user types the command.  If set to a nonzero value other than 1, a
+	split is open which displays the lines where the search matches. The
+	replacement text in the split is hightlighted using
+	|hl-IncSubstitute|. Note: Only '/' is supported as a delimiter.
```

Comment:
This is not a number option, number options define quantity (unless they are legacy like &laststatus (which BTW rejects any values, but 0, 1, 2)). I suggest string with values

1. Empty (default): 0.
2. `"origin"`: 1.
3. `"splitview"`: 2.